### ISSUE TITLE
NOT install the requirements

### ISSUE THREAD

**Issue by alexdo83** [robinkbr/automated_tiktok_video_creation#1]
i tried to install requirements.txt library, but can't install, can you give me advice?
![image](https://user-images.githubusercontent.com/25850343/210138876-ecfd604e-14e9-4aed-ab85-b58bd72a97d3.png)

**Comment by xMunduSCodEx** [comment]
Inside requirements.txt some packages point to a local file that because you get this error.
remove "@ file:///tmp/build/80754af9/aiosignal_1637843061372/work" or similar string inside requirements and keep only the package name. anyway I'm facing issue to install the requirements as well as they point to old packages so the better way is to install them manually

**Comment by xMunduSCodEx** [comment]
I've been able to solve with these requirements:
`absl-py==1.2.0
aiohttp
aiosignal
alabaster
anaconda-client
anaconda-navigator
appdirs==1.4.4
applaunchservices
appnope
appscript
argon2-cffi
argon2-cffi-bindings
arrow
astroid
asttokens
astunparse==1.6.3
async-generator==1.10
async-timeout
atomicwrites==1.4.0
attrs
audiotsm==0.1.2
Automat
autopep8
Babel
backcall
backports.functools-lru-cache
backports.tempfile
backports.weakref==1.0.post1
bcrypt
beautifulsoup4
binaryornot
bitarray
black==19.10b0
bleach
boto3
botocore
brotlipy==0.7.0
bs4==0.0.1
cachetools
certifi
cffi
chardet
charset-normalizer
chex==0.1.3
click
clock==0.1
cloudpickle
clyent==1.2.2
colorama
colorcet
commonmark==0.9.1
conda==4.3.13
conda-build==3.21.9
conda-content-trust
conda-pack
conda-package-handling
conda-repo-cli==1.0.5
conda-token
conda-verify==3.4.2
constantly==15.1.0
cookiecutter
cryptography
cssselect==1.1.0
cvzone==1.5.6
cycler
Cython
cytoolz==0.11.0
dask
debugpy
decorator
defusedxml
diff-match-patch
distributed
dm-tree==0.1.7
docutils
entrypoints
et-xmlfile==1.1.0
etils==0.6.0
executing
fake-useragent==0.1.11
fastjsonschema
feedfinder2==0.0.4
feedparser==6.0.10
ffmpeg-python==0.2.0
filelock
flake8
Flask
flatbuffers==1.12
flax==0.5.3
fonttools==4.25.0
frozenlist
fsspec
future
gast==0.4.0
glob2
gmpy2
google-api-core==2.8.2
google-api-python-client==2.55.0
google-auth
google-auth-httplib2==0.1.0
google-auth-oauthlib==0.4.6
google-cloud-core
google-cloud-storage
google-crc32c
google-pasta==0.2.0
google-resumable-media
googleapis-common-protos==1.56.4
greenlet
grpcio
h11==0.13.0
HeapDict
html2text==2020.1.16
httplib2==0.20.4
huggingface-hub==0.8.1
hyperlink
idna
imageio==2.21.0
imageio-ffmpeg==0.4.7
imagesize
importlib-metadata
importlib-resources==5.9.0
incremental
isodate==0.6.0
itsdangerous==2.1.0
jax
jaxlib==0.3.15
jdcal
jedi
jieba3k==0.35.1
Jinja2
jinja2-time
jmespath
joblib
json5
jsonschema
jupyter
jupyter-client
jupyter-console
jupyter-core
jupyter-server
jupyterlab
jupyterlab-pygments
jupyterlab-server
jupyterlab-widgets
keras==2.9.0
Keras-Preprocessing==1.1.2
keyring
kiwisolver
lazy-object-proxy
libarchive-c
libclang==14.0.1
llvmlite==0.38.0
locket
lxml
Markdown
MarkupSafe
matplotlib-inline
mccabe==0.6.1
mediapipe==0.8.10.1
mistune
mkl-fft==1.3.1
mkl-random
mkl-service==2.4.0
mock
monty==2022.4.26
moviepy==1.0.3
mpmath==1.2.1
msgpack
multidict
multipledispatch
munkres==1.1.4
mypy-extensions==0.4.3
navigator-updater
nbclassic
nbclient
nbconvert
nbformat
nest-asyncio
networkx
newspaper3k==0.2.8
nltk
nose
notebook
numpy
numpydoc
oauth2client==4.1.3
oauthlib==3.2.0
olefile
openai==0.20.0
opencv-contrib-python==4.6.0.66
opencv-python==4.6.0.66
openpyxl
opt-einsum==3.3.0
optax==0.1.3
outcome==1.2.0
packaging
palettable==3.3.0
pandas==1.4.3
pandas-stubs==1.4.3.220718
pandocfilters
param
parse==1.19.0
parsel
parso
partd
pathlib
pathspec==0.7.0
pep8==1.7.1
pexpect
pickleshare
Pillow==9.2.0
pkginfo
plotly
pluggy
poyo
proglog==0.1.10
prometheus-client
prompt-toolkit
Protego
protobuf==3.20.1
psutil
ptyprocess
pure-eval
py
pyasn1
pyasn1-modules==0.2.8
pycodestyle
pycosat==0.6.3
pycparser
pyct
pycurl==7.44.1
PyDispatcher==2.0.5
pydocstyle
PyDrive==1.3.1
pyee==8.2.2
pyflakes
Pygments
PyHamcrest
PyJWT
pylint
pyls-spyder==0.4.0
pymongo==4.2.0

pyodbc
pyOpenSSL
pyparsing
pyppeteer==1.0.2
pyquery==1.4.3
pyrsistent
PySocks
pytest==7.1.1
python-dateutil
python-lsp-black
python-lsp-jsonrpc==1.0.0
python-lsp-server==1.2.4
python-slugify
python-snappy
pytube==12.1.0
pytube3==9.6.4
pytube4==0.1.1
pytz
pytz-deprecation-shim==0.1.0.post0
pyviz-comms
PyYAML==6.0
pyzmq
QDarkStyle
qstylizer
QtAwesome
qtconsole
QtPy
queuelib==1.5.0
rake-nltk==1.0.6
regex
requests
requests-file
requests-html==0.10.0
requests-oauthlib==1.3.1
rich==11.2.0
rope
rsa
Rtree
ruamel-yaml-conda
ruamel.yaml==0.17.21
ruamel.yaml.clib==0.2.6
s3transfer
scipy==1.9.0
Scrapy
selenium==4.3.0
Send2Trash
service-identity
sgmllib3k==1.0.0
sip==4.19.13
six
smart-open
sniffio
snowballstemmer
sortedcollections
sortedcontainers
soupsieve
spglib==1.16.5
Sphinx
sphinxcontrib-applehelp
sphinxcontrib-devhelp
sphinxcontrib-htmlhelp
sphinxcontrib-jsmath
sphinxcontrib-qthelp
sphinxcontrib-serializinghtml
spyder
spyder-kernels
SQLAlchemy
stack-data
sympy
tabulate==0.8.9
TBB==0.2
tblib
tenacity
tensorboard==2.9.1
tensorboard-data-server==0.6.1
tensorboard-plugin-wit==1.8.1
tensorflow==2.9.1
tensorflow-estimator==2.9.0
tensorflow-io-gcs-filesystem==0.26.0
tensorstore==0.1.21
termcolor==1.1.0
terminado
testpath
text-unidecode
textdistance
threadpoolctl
three-merge
tinycss
tinysegmenter==0.3
tldextract
tokenizers==0.12.1
toml
tomli
toolz
torch==1.13.0.dev20220731
torchaudio==0.13.0.dev20220731
torchvision==0.14.0.dev20220731
tornado
tqdm
traitlets
transformers==4.21.0
trio==0.21.0
trio-websocket==0.9.2
Twisted
typed-ast
types-pytz==2022.1.2
typing_extensions
tzdata==2022.1
tzlocal==4.2
ujson
Unidecode
uritemplate==4.1.1
urllib3
w3lib
watchdog
wcwidth
websocket-client==1.2.1
websockets==10.0
Werkzeug==2.1.1
widgetsnbextension
wrapt
wsproto==1.1.0
wurlitzer
xlrd
XlsxWriter
xlwings==0.24.9
yapf
yarl
zict==2.0.0
zipp
zope.interface`

https://pastebin.com/CcJEL1fP
